webapp/src/pages/boardPage.tsx

```diff
@@ -220,12 +224,14 @@ const BoardPage = (props: Props): JSX.Element => {
             if (timeout) {
                 clearTimeout(timeout)
             }
-            wsClient.unsubscribeToWorkspace(match.params.workspaceId || '0')
+            if (subscribedToWorkspace) {
+                wsClient.unsubscribeToWorkspace(match.params.workspaceId || '0')
+            }
             wsClient.removeOnChange(incrementalUpdate)
             wsClient.removeOnReconnect(() => dispatch(loadAction(match.params.boardId)))
             wsClient.removeOnStateChange(updateWebsocketState)
         }
-    }, [match.params.workspaceId, props.readonly])
+    }, [match.params.workspaceId, props.readonly, match.params.boardId])
```

Comment:
Do the changes in this component related to this PR? Seems not but please clarify.

Reply:
Yes, this fixes some race condition with websocket initialization. Cypress clicks too fast and sometimes websocket is still not connected during this process and tests are failing.